pandatool/src/deploy-stub/deploy-stub.c

```diff
@@ -640,8 +694,21 @@ int main(int argc, char *argv[]) {
   ModuleDef *moddef;
   const char *log_filename;
   void *blob = NULL;
+#if defined(MS_WINDOWS)
```

Comment:
Do we really need an ifdef here? Maybe we already have a platform independent define somewhere for this?